aggregate-install.rst

```diff
@@ -459,135 +459,144 @@ To run on ODK Aggregate on a Tomcat server backed with a MySQL or PostgreSQL dat
 Installing on AWS (Cloud)
 --------------------------
 
-Following are basic details for setting up ODK Aggregate to run on a Linux micro-instance on the Amazon Web Services EC2 infrastructure.
+Following are basic details on setting up ODK Aggregate to run on a Linux micro-instance on the Amazon Web Services EC2 infrastructure. 
+For screenshots and more on the general set-up of Tomcat on AWS, see the excellent three-part `Cat in the cloud Apache Tomcat Series <http://www.excelsior-usa.com/articles/tomcat-amazon-ec2-basic.html>`_. `Amazon’s getting-started guides <https://aws.amazon.com/documentation/gettingstarted/>`_ are also quite helpful.
+
+- First, sign up for Amazon Web Services EC2 at `<http://aws.amazon.com/ec2/>`_.
```

Comment:
This hasn't been added to the contrib guide, but - we've decided that multi-step procedures should use ordered lists (numbered). Please update to use the new stye.

Content included as part of the step (explanations, images, etc) should be indented (three-spaces) to become part of the List Item:

```
1. Do this thing.
   
   .. image:: img-of-thing.*

   Also some explanation.

```

This will cause everything after the first sentence to get wrapped in a `details` block and hidden behind a toggle. (See recently added collect user guide pages for live examples)

Reply:
Shall I do this for the complete install doc in this PR itself?

Reply:
yes.